content/documentation/multiformat-output.md

```diff
@@ -1,12 +1,218 @@
+
+1. Navigate to your project and compress the `site` folder.
+
+2. Go to Netlify Drop: [https://app.netlify.com/drop](https://app.netlify.com/drop). Make sure you are logged in to your account and then drag-and-drop your compressed `site` folder into the indicated area.
+
+3. You will be given a default URL to preview your project. Rename this URL by navigating to “Site settings” and changing the site name. (You also have the option to buy a domain or set-up a domain you already own.)
+
+If you make further edits to your project and would like to preview them you will need to repeat this process.
+
+1. Delete the old compressed `site` folder.
+
+2. Run `quire site` again (your files will be automatically overwritten.)
+
+3. Compress the newly updated `site` folder.
+
+4. In Netlify, navigate to “Deploys” at the top of the page. You will see a blank space that reads, “Need to update your site." Simply drag-and-drop your new compressed `site` folder here and your link will be automatically updated.
+
+### Continuous Deploy with Netlify
+
+By linking GitHub directly to your Netlify account, any time you merge changes in Github, your preview link will be automatically updated. This process requires a few extra steps to get set-up, but will save time in the long run.
+
+1. If you haven't already, create a repository for your project on GitHub. You can find directions in the [*GitHub*](/documentation/gitub) section of our documentation.
+
+2. To proceed with deployment, you will need to add two files to your repo: `netlify.toml` and `package.json`. Download them [here](/downloads/site-deploy.zip), add them to your repo, and merge changes before proceeding.
+
+3. Log in to [Netlify](https://app.netlify.com/). On the “Team Overview” page, click the button that says “New Site from Git.”
+
+4. Next you'll want to “Connect to Git Provider.” Choose either GitHub or GitLab.
+
+5. If you see “no repositories found” you will be prompted to configure Netlify on GitHub.
+
+6. Once this configuration is complete, choose the repository you would like to preview.
+
+7. Set the "build command" as `npm run build` and the "publish directory" as `site`.  
+
+8. Hit "Deploy Site." Depending on the size of your project, this may take a few moments. Follow along with the build process (and check for errors) by navigating to "Publication Deploys" section and scrolling down to the "Deploy Log".
+
+9. You will be given a default URL to preview your project. You can rename this URL by navigating to “Site settings” and changing the site name. (You also have the option to buy a domain or set-up a domain you already own.)
+
+10. Should you need to make any updates to your site, simply merge the changes and Netlify will automatically update your preview link. You can check "Production Deploys" in the site overview section to these track changes.
```

Comment:
Should we inlcude a link somehwere to Netlify's own instructions about this?

Also, these instructions assume a fairly solid understanding of git and github, using words like merge and pull request. Should this set of instructions come with something of a warning about this?

Reply:
Audrey pointed this out as well. Specifically the phrases "merge changes" and "add to your repo". What if we linked the terms to the GitHub glossary (something we should maybe make more use of in general)? For example, https://docs.github.com/en/github/getting-started-with-github/github-glossary#pull-request. 

Audrey, also got tripped up on the term "compress" when discussing the site folder. Not sure if saying "make a zip file" would be better.

Reply:
On second thought, while I really like the glossary idea, as Audrey noted, it's a pain clicking on links because they don't open in a new window. Related to a similar issue Tilt West was having https://github.com/tiltwest/tilt-vol-2/issues/10. I could have sworn I made an issue for this in the Quire repo.